client/reducers/usersReducer.js

```diff
@@ -0,0 +1,11 @@
+import * as ActionTypes from '../actions/actionTypes';
+
+export default (state = {}, action) => {
+    switch (action.type) {
+        case ActionTypes.LOAD_USERS:
+            return action.payload;
+
+        default:
+            return state;
+    }
+}
```

Comment:
this reducer handles the users. the users are fetched when the `asyncAction` is dispatched and the results are passed into this reducer which takes care of loading it into the state. the fetched users have already been paginated and filtered on the backend